src/index.ts

```diff
@@ -21,8 +21,15 @@ export interface PyroscopeConfig {
   configured: boolean
 }
 
-const INTERVAL = 10000
-const SAMPLERATE = 100
+// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L37-L38
+// The Interval in which samples should be collected. (in microseconds)
+// converting to microseconds
+const SAMPLE_RATE =  Number(process.env['PYROSCOPE_SAMPLE_RATE'])*10 || 10000 // in microseconds -> ex. 10000 microseconds --> 0.01 seconds --> 100Hz
```

Comment:
Should we maybe keep `SAMPLE_RATE` in milliseconds and only convert to microseconds when pass it to the profiler constructor (`intervalMicros`)?

```js
// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L37-L38
const SAMPLE_RATE =  Number(process.env['PYROSCOPE_SAMPLE_RATE']) || 10
```

Also, I would consider renaming `SAMPLE_RATE` to `SAMPLING_INTERVAL` to make it consistent with the profiler parameters (same for env var).

Reply:
Sure, but for environment variable, Should you keep `PYROSCOPE_SAMPLE_RATE `? 
As in our documentation we suggest:

```
Environment variables MUST have PYROSCOPE_ prefix and be in UPPER_SNAKE_CASE format, for example:
```
and for sampling rate, we have a config  -> `sample-rate`
https://pyroscope.io/docs/agent-configuration-exec/#environment-variables